src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const filteredArray = sourceString
```

Comment:
The initial split operation on the `sourceString` is correct, as it divides the string into an array of style declarations based on semicolons.